dev/packages/alpha/system/0.0.1/dataset/diskio/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1 @@
+metricsets: ["diskio"]
```

Comment:
Do we miss period here as a config option? And also some other datasets?

Reply:
`period` is at the module level, if that's what you mean?

Reply:
No, there are two things here. First I assume this stream template should also contain an entry for the period like few other datasets? The second part is if it is a global option on the package level or per dataset. I think it should be per dataset and you have it specified in some of the datasets already. Important is that it can only be either on the package or the dataset level, not both.